File: Phyton/Product.py
# Produk merupakan kelas orang tua tertinggi dengan urutan
# Produk <- Hardware <- Memory

class Product:

    #  konstruktor
    def __init__(self):
        self.price = 0
        self.idProduct = "blank"

    # semua metode set dan gett disetiap atribut

    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price 

    def setIdProduct(self, idProduct):
        self.idProduct = idProduct

    def getIdProduct(self):
        return self.idProduct 

    

File: Java/Product.java
// Produk merupakan kelas orang tua tertinggi dengan urutan
// Produk <- Hardware <- Memory

public class Product {
    // deklarasi atribut
    private int price = 0;
    private String idProduct = "";

    // parameter konstruktor
    public Product(int price, String idProduct) {
        this.price = price;
        this.idProduct = idProduct;
    }

    // Konstruktor
    public Product() {

    }

    // Methods Set dan get seluruh atribut ..

    public void setPrice(int price) {
        this.price = price;        
    }
    public int getPrice() {
        return this.price;        
    }

    public void setIdProduct(String idPString) {
        this.idProduct = idProduct;
    }
    public String getIdProduct() {
        return this.idProduct;
    }
}